packages/mobile/src/components/usdc-manual-transfer-drawer/USDCManualTransferDrawer.tsx

```diff
@@ -96,6 +98,12 @@ export const USDCManualTransferDrawer = () => {
   const handleConfirmPress = useCallback(() => {
     Clipboard.setString(USDCUserBank ?? '')
     toast({ content: messages.copied, type: 'info' })
+    trackEvent(
+      make({
+        eventName: Name.PURCHASE_CONTENT_USDC_USER_BANK_COPIED,
+        address: USDCUserBank ?? ''
```

Comment:
is it helpful at all to track the empty strings? maybe the tracking of this event should be in an if statement that makes sure address is truthy

Reply:
spoke offline, i think we actually want to track the empty string case bc that would indicate a bug.. theoretically should never get an empty string.